semanticdb/scalac/library/src/main/scala/scala/meta/internal/semanticdb/scalac/TextDocumentOps.scala

```diff
@@ -604,6 +604,13 @@ trait TextDocumentOps { self: SemanticdbOps =>
               case _ =>
             }
 
+            def isClassOf(pos: m.Position): Boolean = {
+              val (chars, classOfChars) = (pos.input.chars, "classOf".toCharArray)
```

Comment:
nit: a more idiomatic way to declare two variables
```suggestion
              val chars = pos.input.chars
              val classOfChars = "classOf".toCharArray
```

Reply:
Fixed.